blockchain/v2/reactor.go

```diff
@@ -493,7 +493,7 @@ func (r *BlockchainReactor) Receive(chID byte, src p2p.Peer, msgBytes []byte) {
 	msg, err := decodeMsg(msgBytes)
 	if err != nil {
 		r.logger.Error("error decoding message",
-			"src", src.ID(), "chId", chID, "msg", msg, "err", err, "bytes", msgBytes)
+			"src", src.ID(), "chId", chID, "msg", msg, "err", err)
```

Comment:
```suggestion
			"src", src.ID(), "chId", chID, "err", err)
```

Reply:
Changes shouldn't be made on backport RPs, they should've been caught on the original PR. If this change is necessary, it should be made against `master` and then backported as well.

Reply:
Also, curious. Why is it OK to not log the msg here, but it is OK to log it upon validation failure?

Reply:
here msg will be `nil` if error is not nil => no need to log it. I would probably remove it from validation too, if it was up to me.

Reply:
I'm going to leave this here for now because, although it's pointless to log `msg` here, it's still safe; and I'm pretty sure these changes should always happen on master first and then trickle over to backport branches. (I'm wary of mixing up our process, given how frequently we seem to get tangled up in it.) But I'll go fix it on master as well.

Reply:
Ahh I see, that makes sense @melekes.

ACK @tessr